megatron/megatron/text_generation/generation_single_thread.py

```diff
@@ -233,8 +226,7 @@ def generate_tokens_probs_and_return_on_first_stage_stream_sub_process(
                 # length, it means we have started generating tokens
                 started = lengths <= context_length
                 # Update the tokens.
-                if started:
-                    tokens[0, context_length] = new_sample[0]
+                tokens[started, context_length] = new_sample[started]
```

Comment:
Why does this pr change these two lines? They are not related to the kunlun chip.